src/aws/credential.rs

```diff
@@ -422,6 +422,17 @@ impl AssumeRoleLoader {
         if let Some(duration_seconds) = &self.config.duration_seconds {
             write!(url, "&DurationSeconds={duration_seconds}")?;
         }
+        if let Some(tags) = &self.config.tags {
+            for (idx, tag) in tags.iter().enumerate() {
+                let parts = tag.split("=").collect::<Vec<&str>>();
```

Comment:
We can use [split_once](https://doc.rust-lang.org/std/primitive.str.html#method.split_once) to avoid an alloc of `Vec<_>`.